docs/source/models/cause.rst

```diff
@@ -146,26 +146,107 @@ What does a model document look like?
 
 [[to be updated based on experience from LTBI cause model document, and generalization thereof]]
 
-Common cause models
+Common Cause Models
 -------------------
 
-.. todo::
-
-   Format as table with model type, description.
-   Fill in descriptions.
+Common cause model structures are described in the table and dicussed in further detail below. `Non-Standard Cause Models`_ are covered afterward.
+
++---------+--------------------------------+---------------------------------------------------------------------+
+|Model    |States                          |Description                                                          |
++=========+================================+=====================================================================+
+|SI_      |Susceptible-Infected            |Simulants never recover from infected (with condition) state         |
+|         |                                |                                                                     |
++---------+--------------------------------+---------------------------------------------------------------------+
+|SIS_     |Susceptible-Infected-Susceptible|Simulants can recover from infected (with condition) state and can   |
+|         |                                |become infected again after recovery                                 |
+|         |                                |                                                                     |
++---------+--------------------------------+---------------------------------------------------------------------+
+|SIR_     |Susceptible-Infected-Recovered  |Simulants can recover from infected (with condition) state and cannot| 
+|         |                                |become infected again after recovery                                 |
+|         |                                |                                                                     |
++---------+--------------------------------+---------------------------------------------------------------------+
+|Neonatal_|With and without condition      |Simulants are born either with or without condition                  |
```

Comment:
I'm currently working on these models and thinking about how to label the model states. What do people think about "F" for "**Free** of Condition" and "C" for "With **Condition**"?

Reply:
I think this could work! I think I remember someone mentioning that dismod has its own naming convention... maybe we could use that for consistency. Does anyone happen to know the abbreviations used in dismod?

Reply:
Abie's book on DisMod-MR uses S for "susceptible" and C for "with condition." But I don't like "susceptible" for the neonatal/congenital disorders because you're never actually susceptible - you're either born with the condition or not.